### ISSUE TITLE
Cannot build docs with Sphinx 2.0.x

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#1205]
(Original issue 1501 created by None on 2019-03-12T13:45:20.802336+00:00)

This is form Sphinx 1.8.x:

```
$ make html
PYTHONPATH=.. sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v1.8.5
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 24 source files that are out of date
updating environment: 24 added, 0 changed, 0 removed
Warning: SmartGameFormatLexer does not have a docstring.                                                                                                                                                                         
reading sources... [100%] languages                                                                                                                                                                                              
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
/home/churchyard/tmp/pygments-main/__venv__/lib64/python3.7/site-packages/sphinx/builders/html.py:1171: RemovedInSphinx20Warning: Now html_sidebars only allows list of sidebar templates as a value. Support for a string value will be removed at Sphinx-2.0.
  self.add_sidebars(pagename, ctx)
writing output... [100%] languages                                                                                                                                                                                               
generating indices... genindex py-modindex
writing additional pages... search
copying static files... done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded.

The HTML pages are in _build/html.

Build finished. The HTML pages are in _build/html.
```

And indeed this is an error with Sphinx 2.0.0b1 (`pip install --pre sphinx`):

```
$ make html
PYTHONPATH=.. sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v2.0.0b1
making output directory... done
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 24 source files that are out of date
updating environment: 24 added, 0 changed, 0 removed
Warning: SmartGameFormatLexer does not have a docstring.                                                                                                                                                                         
reading sources... [100%] languages                                                                                                                                                                                              
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [  4%] docs/api                                                                                                                                                                                                
Theme error:
An error happened in rendering the page docs/api.
Reason: TemplateNotFound()
```

**Comment by Anteru** [comment]
(Original issue 1501 last updated on 2019-04-02T12:25:04.326155+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by Anteru on 2019-04-02T12:24:49.837190+00:00)

Fix documentation build using Sphinx 2.0 (fixes #1501.)

→ <<cset 113118f16792>>

**Comment by Anteru** [comment]
(Original comment by Anteru on 2019-04-02T12:25:04.265991+00:00)

Thanks for the report!
